mysql-test/suite/tianmu/t/alter_table.test

```diff
@@ -0,0 +1,47 @@
+#
+#ALTER TABLE table_name MODIFY ...
+#
+
+CREATE TABLE `t_test` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `first_name` varchar(10),
+  `last_name` varchar(10),
+  `sex` varchar(5),
+  `score` int(11),
+  `copy_id` int(11),
+  PRIMARY KEY (`id`)
+) ENGINE=tianmu DEFAULT CHARSET=utf8;
```

Comment:
Checking the engine. 
Pls. refer to : https://www.cnblogs.com/sms-xc/p/3651901.html